Update Slack integration prereqs (#4856)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/job-notifications.md

### Prerequisites 
- You must be an administrator of the Slack workspace. 
- You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).

> - The notification channel must be a public channel. The integration only supports public channels in the Slack workspace. 

Author: ```suggestion
- The notification channel must be a public channel. The integration only supports public channels that are in the Slack workspace. 
```

> - The notification channel must be a public channel. The integration only supports public channels that are in the Slack workspace. 

Author: ```suggestion
- Make sure the notification channel is a public channel. The integration only supports public channels that are in the Slack workspace. 
```

> - Make sure the notification channel is a public channel. The integration only supports public channels that are in the Slack workspace. 

Author: ```suggestion
- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 
```

> - Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 

Reviewer: We could probably cut this down to just
```suggestion
- The integration only supports public channels in the Slack workspace. 
```
